src/main/java/com/google/api/codegen/viewmodel/LongRunningOperationDetailView.java

```diff
@@ -21,6 +21,8 @@
 
   public abstract String operationReturnType();
 
+  public abstract String operationResponseType();
```

Comment:
When I was working on this, my thoughts were that
- "return type" -> The type returned by the client's method (`OperationFuture<Foo>` in Java and `FooOperation` in Go). I was tempted to rename this `handleType` but that sound just as vague.
- "response type" -> The concrete type of the message in [Operation.result.response field](https://github.com/googleapis/googleapis/blob/master/google/longrunning/operations.proto#L112).